pkg/fab/discovery/discovery.go

```diff
@@ -62,6 +62,14 @@ type Response interface {
 	Error() error
 }
 
+// WithIndifferentFilter returns NoPriorities/NoExclusion filter.
+// Note: this method was added just to allow users of Client to be able to use Endorsers method in response, which requires Filter as an argument.
+// It's impossible to implement interface because Filter is placed under internal dir which is not available to end user.
+// A user should filter peers by himself.
+func WithIndifferentFilter() discclient.Filter {
```

Comment:
We usually use the `With` Prefix for options.
@bstasyszyn thoughts?

Reply:
I thought about that too, IndifferentFilter() fits better, doesn't it?

Reply:
As it's creating a filter object, I think a `New` or `Create` prefix might be applicable.

Reply:
I think it's simpler to do this:

var NoFilter = discclient.NoFilter

Reply:
Actually, I think that it's considered bad practice to have a global var. So New prefix would be good, as Troy suggests.

Reply:
done